.vscode/cspell.json

```diff
@@ -504,6 +504,29 @@
         "undoc"
       ]
     },
+    {
+      "filename": "sdk/storage/azure-storage-file-datalake/**",
+      "words": ["myfile", "dlake"]
+    },
+    {
+      "filename": "sdk/storage/azure-storage-file-datalake/**/*.py",
+      "words": [
+        "rwxrwxrwx", "rwxrw", "rwxr", "ABFS", "deletionid", "saoid", "testfs",
+        "mynewfilesystems", "myleasefilesystem", "mymetadatafilesystemsync",
+        "mypathfilesystem", "myclientfilesystem", "mydirectoryfilesystem", "myfilesystemforgetclient",
+        "mynewdirectory", "mydirectory", "mynewfile", "myfilesystem", "mynewfilesystem",
+        "myfilesystemforlistpaths", "cpkfile", "dont", "fsname", "fpath", "dname", "racwdlmeop",
+        "filesys", "nonexistentfs", "testid", "cpkdirectory", "cpksubdirectory"
+      ]
+    },
+    {
+      "filename": "sdk/storage/azure-storage-file-datalake/**/*.md",
+      "words": ["thead", "tbody", "myfilesystem"]
+    },
+    {
+      "filename": "sdk/storage/**/_shared/*.py",
+      "words": ["yarl", "mday"]
```

Comment:
I think after #25923 you shouldn't need to add anything for `_shared`. Please merge from `main` and see if that eliminates the need for this.

Reply:
Yes, my friend @KdHyeon0661 has done a great job!